### PR TITLE
Consistency in "New" buttons on tables

### PR DESCRIPTION
**Pull request by Ibochkarev**
### What does it do?
I corrected the translations of the buttons for consistency:
-Access Controls
---User Groups: "New User Group" -> "New Group"
---Roles: "Create New" -> "New Role"
---Access Policies: "Add" -> "New Policy"
-Resource Groups: "Create Resource Group" -> "New Group"
-Property Sets
---Property Sets: "New Property Set" -> "New Set"
---Properties: "Create Property" -> "New Property" Under System
-Contexts: "Create New" -> "New Context"
-Actions
---Top Menu: "Create Menu" -> "New Menu" 
-Package Management
---Packages: "Add New Package" -> "New Package" 
---Providers: "Add New Provider" -> "New Provider" 
-Content Types: "New Content Type" -> "New Type" 
-Namespaces: "Create New" -> "New Namespace"
-System Settings: "Create New Setting" -> "New Setting"

I did not find the location of these buttons:
-Form Customization: "Add" -> "New Rule" Under Tools
-Lexicon Management
---Languages: "Create Language" -> "New Language"

### Why is it needed?
To improve consistency within the control panel

### Related issue(s)/PR(s)
#1188

### CODE REVIEW

File: core/lexicon/en/access.inc.php [hunk 1]
```diff
@@ -81,7 +81,7 @@
 $_lang['resource_group_add'] = 'Add Resource Group';
 $_lang['resource_group_access_remove'] = 'Remove Resource from Group';
 $_lang['resource_group_access_remove_confirm'] = 'Are you sure you want to remove this resource from the resource group?';
-$_lang['resource_group_create'] = 'Create Resource Group';
+$_lang['resource_group_create'] = 'New Group';
```
Comment (Jako) [inline core/lexicon/en/access.inc.php]: I prefer `New Resource Group`

File: core/lexicon/en/content_type.inc.php [hunk 2]
```diff
@@ -22,7 +22,7 @@
 $_lang['content_type_main_tab'] = 'Main';
 $_lang['content_type_header_tab'] = 'Custom headers';
 $_lang['content_type_header_title'] = 'Create/edit header';
-$_lang['content_type_new'] = 'New Content Type';
+$_lang['content_type_new'] = 'New Type';
```
Comment (Jako) [inline core/lexicon/en/content_type.inc.php]: I prefer `New Content Type`

File: core/lexicon/en/policy.inc.php [hunk 3]
```diff
@@ -25,7 +25,7 @@
 $_lang['permissions_desc'] = 'Here you can define specific permissions that this policy will contain. All user groups with this policy will inherit these permissions.';
 $_lang['policies'] = 'Access Policies';
 $_lang['policy'] = 'Access Policy';
-$_lang['policy_create'] = 'Create Access Policy';
+$_lang['policy_create'] = 'New Policy';
```
Comment (Jako) [inline core/lexicon/en/policy.inc.php]: I prefer `New Access Policy`

File: core/lexicon/en/propertyset.inc.php [hunk 4]
```diff
@@ -66,7 +66,7 @@
 $_lang['propertyset_err_ns_name'] = 'Please specify a name for the property set.';
 $_lang['propertyset_err_remove'] = 'An error occurred while trying to remove the property set.';
 $_lang['propertyset_err_update'] = 'An error occurred while trying to update the property set.';
-$_lang['propertyset_new'] = 'New Property Set';
+$_lang['propertyset_new'] = 'New Set';
```
Comment (Jako) [inline core/lexicon/en/propertyset.inc.php]: I prefer `New Property Set`

File: core/lexicon/en/user.inc.php [hunk 5]
```diff
@@ -141,7 +141,7 @@
 $_lang['user_group_member_err_nf'] = 'User is not in this user group!';
 $_lang['user_group_member_err_remove'] = 'An error occurred while trying to remove the user from the group.';
 $_lang['user_group_member_err_save'] = 'An error occurred while trying to add the user to the group.';
-$_lang['user_group_new'] = 'New User Group';
+$_lang['user_group_new'] = 'New Group';
```
Comment (Jako) [inline core/lexicon/en/user.inc.php]: I prefer `New User Group`

File: manager/assets/modext/widgets/security/modx.grid.role.js [hunk 6]
```diff
@@ -44,7 +44,7 @@ MODx.grid.Role = function(config) {
             ,sortable: true
         }]
         ,tbar: [{
-            text: _('create_new')
+            text: _('new_role')
```
Comment (Jako) [inline manager/assets/modext/widgets/security/modx.grid.role.js:47]: You can't rename the lexicon key here without renaming it in the lexicon files.

File: manager/assets/modext/widgets/system/modx.grid.context.js [hunk 7]
```diff
@@ -46,7 +46,7 @@ MODx.grid.Context = function(config) {
             ,editor: { xtype: 'numberfield' }
         }]
         ,tbar: [{
-            text: _('create_new')
+            text: _('context_create')
```
Comment (Jako) [inline manager/assets/modext/widgets/system/modx.grid.context.js:49]: You can't rename the lexicon key here without renaming it in the lexicon files.

File: manager/assets/modext/workspace/namespace/modx.namespace.panel.js [hunk 8]
```diff
@@ -77,7 +77,7 @@ MODx.grid.Namespace = function(config) {
             ,editor: { xtype: 'textfield' }
         }]
         ,tbar: [{
-            text: _('create_new')
+            text: _('namespace_create')
```
Comment (Jako) [inline manager/assets/modext/workspace/namespace/modx.namespace.panel.js:80]: You can't rename the lexicon key here without renaming it in the lexicon files.

**Review by Jako — CHANGES_REQUESTED** [review]

**Comment by Jako** [comment]
It is nice to have consistent usage of new or create, but IMO this PR removes too much.

File: manager/assets/modext/widgets/security/modx.grid.role.js (hunk 6 above)
Comment (Ibochkarev) [reply manager/assets/modext/widgets/security/modx.grid.role.js:47]: This entry is on the way `core/lexicon/en/default.inc.php` - in 279 line. But she was not used

File: manager/assets/modext/widgets/system/modx.grid.context.js (hunk 7 above)
Comment (Ibochkarev) [reply manager/assets/modext/widgets/system/modx.grid.context.js:49]: This entry is on the way `core/lexicon/en/context.inc.php` - in 11 line. But she was not used

File: manager/assets/modext/workspace/namespace/modx.namespace.panel.js (hunk 8 above)
Comment (Ibochkarev) [reply manager/assets/modext/workspace/namespace/modx.namespace.panel.js:80]: This entry is on the way `core/lexicon/en/propertyset.inc.php` - in 32 line. But she was not used

**Comment by Ruslan-Aleev** [comment]
In MODX there are similar in meaning, and sometimes identical lines, but different keys in translation.
For example, **new_tv | tv_new | add_tv**
I have collected some duplicate (lines **new, create and add**, others did not look), we need to look at and delete the extra lines - [lang_double.txt](https://github.com/modxcms/revolution/files/2799942/lang_double.txt)

File: core/lexicon/en/propertyset.inc.php [hunk 9]
```diff
@@ -29,7 +29,7 @@
 $_lang['properties_remove_confirm'] = 'Are you sure you want to delete these properties?';
 $_lang['property'] = 'Property';
 $_lang['property_area_desc'] = 'The area, or group, this property belongs to.';
-$_lang['property_create'] = 'Create Property';
+$_lang['property_create'] = 'New Property Set';
```
Comment (Jako) [inline core/lexicon/en/propertyset.inc.php]: There is a difference between `Property` and `Property Set`

File: core/lexicon/en/propertyset.inc.php (hunk 4 above)
Comment (Jako) [inline core/lexicon/en/propertyset.inc.php]: There is a difference between `Property` and `Property Set`

File: manager/assets/modext/widgets/security/modx.grid.role.js (hunk 6 above)
Comment (Jako) [reply manager/assets/modext/widgets/security/modx.grid.role.js:47]: Ok, thanks.

**Comment by Ibochkarev** [comment]
@Jako Hi! Is it okay?

**Review by Jako — APPROVED** [review]

**Review by Mark-H — APPROVED** [review]

**Pull request merged**